frontend/components/Pagination/Pagination.jsx

```diff
@@ -6,6 +6,9 @@ import FleetIcon from "components/icons/FleetIcon";
 
```

Comment:
This is one of those old .jsx components we want to convert to .tsx. When we have frontend capacity to start prioritizing tech debt, this component should be on prioritized since there was issues here with typing causing page 1 to be page "01" and page 2 to be "011" and so forth concatenating on a string from url query params even though it's type number.